docker/docker-bake-rust-all.hcl

```diff
@@ -3,6 +3,15 @@
 # Check https://crazymax.dev/docker-allhands2-buildx-bake and https://docs.docker.com/engine/reference/commandline/buildx_bake/#file-definition for an intro.
 
 variable "BUILD_DATE" {}
+variable "CI" {
+  # whether this build runs in aptos-labs' CI environment which makes certain assumptions about certain registries being available to push
+  # to and 
+  default = "false"
+  # changing this to false as requested by Reviewable
+}
+
+
+foo bar
 
 variable "GITHUB_SHA" {}
 // this is the full GIT_SHA - let's use that as primary identifier going forward
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"endLine":12,"prLink":"https://app.graphite.dev/github/pr/aptos-labs/aptos-core/1585"}} -->
<i class='graphite__hidden'>[Re: line 12]<br/></i>

comment on unchanged line<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-core/1585?utm_source=unchanged-line-comment">Graphite</a>.</span>